src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.cs

```diff
@@ -0,0 +1,297 @@
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void SingleVariableDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int P1 { get; set; }
+    static void Main(string[] args)
+    {
+        /*<bind>*/int i1;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: System.Int32 i1 (OperationKind.VariableDeclaration)
+";
+            VerifyOperationTreeForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void MultipleDeclarations()
+        {
+            string source = @"
+class Program
+{
+    static void Main(string[] args)
+    {
+        /*<bind>*/int i1, i2;/*</bind>*/
+    }
+}
+";
+            string expectedOperationTree = @"
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: System.Int32 i1 (OperationKind.VariableDeclaration)
+  IVariableDeclaration: System.Int32 i2 (OperationKind.VariableDeclaration)
+";
+            VerifyOperationTreeForTest<LocalDeclarationStatementSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void FixedStatementDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int i1;
+    static void Main(string[] args)
+    {
+        var reference = new Program();
+        unsafe
+        {
+            fixed (/*<bind>*/int* p = &reference.i1/*</bind>*/)
+            {
+
+            }
+        }
+    }
+}
+";
+            string expectedOperationTree = @"
+IFixedStatement (OperationKind.FixedStatement)
+  IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+    IVariableDeclaration: System.Int32* p (OperationKind.VariableDeclaration)
+      Initializer: IAddressOfExpression (OperationKind.AddressOfExpression, Type: System.Int32*)
+          IFieldReferenceExpression: System.Int32 Program.i1 (OperationKind.FieldReferenceExpression, Type: System.Int32)
+            Instance Receiver: ILocalReferenceExpression: reference (OperationKind.LocalReferenceExpression, Type: Program)
+  IBlockStatement (0 statements) (OperationKind.BlockStatement)
+";
+            VerifyOperationTreeForTest<VariableDeclarationSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")]
+        public void FixedStatementMultipleDeclaration()
+        {
+            string source = @"
+class Program
+{
+    int i1, i2;
+    static void Main(string[] args)
+    {
+        var reference = new Program();
+        unsafe
+        {
+            fixed (/*<bind>*/int* p1 = &reference.i1, p2 = &reference.i2/*</bind>*/)
```

Comment:
>&reference.i1 [](start = 39, length = 13)

Consider also test with the value and with equals value missing (two different scenarios).